docs/reference/appointment-types.md

```diff
@@ -0,0 +1,1192 @@
+        "event_hide_decline": false,
+        "hide_places": false,
+        "hide_support": true,
+        "hide_title": true,
+        "forbid_add_places": true,
+        "min_dates": 1,
+        "one_slot": true,
+        "auto_confirm": true,
+        "invite_title": "Book a short appointment with",
+        "redirect_url": "https://www.example.com/thanks-for-booking",
+        "fixed_places": [
+          {
+            "name": "Office",
+            "address": "Office address"
+          },
+          {
+            "name": "Phone",
+            "address": "0102030405"
+          }
+        ]
+      }
+    }'
+```
+
+> RESPONSE SAMPLE
+
+```json light-code
+{
+      "name": "Lunch or dinner",
+      "handle": "lunch-or-dinner",
+      "message": "Welcome to my VERY NEW page",
+      "active": true,
+      "availability": "5f1b018dc1ac5dc46efc64b8"
+      "public": true
+      "description": "My ANOTHER appointment type"
+      "custom": {
+        "auto_message": "Thank you for your booking. See you soon IN 2021.",
+        "auto_title": "RDV {{invitee}} / {{me}}",
+        "ask_phone": true,
+        "ask_company": false,
+        "block_new_invitee": false,
+        "duration": 30,
+        "set_lang": true,
+        "enable_api": true,
+        "fixed_lang": "en",
+        "set_timezone": true,
+        "fixed_timezone": "Europe/London",
+        "event_hide_decline": false,
+        "hide_places": false,
+        "hide_support": true,
+        "hide_title": true,
+        "forbid_add_places": true,
+        "min_dates": 1,
+        "one_slot": true,
+        "auto_confirm": true,
+        "invite_title": "Book a short appointment with",
+        "redirect_url": "https://www.example.com/thanks-for-booking",
+        "fixed_places": [
+          {
+            "name": "Office",
+            "address": "Office address"
+          },
+          {
+            "name": "Phone",
+            "address": "0102030405"
+          }
+    ]
+  }
+}
+```
+
+:::::
+
+::::::
+
+## Delete an Appointment type
+
+::::: panel
+:::: left
+
+> ENDPOINT <small>Authorization `apiKey`</small>
+
+```http
+DELETE /v2/vytemes/:nickname/types/:handle HTTP/1.1
+```
+
+<attributes title="Path parameters">
+  <attribute name="nickname" type="string">
+
+The nickname of the Vyte Page.
+
+  </attribute>
+  <attribute name="handle" type="string">
+    
+The handle of the Appointment type (urlified from the name par default).
+
+  </attribute>
+</attributes>
+
+<attributes title="Query parameters" :isEmpty=true />
+
+<returns title="Returns">
+
+Returns an object containing the number of row affected and the status if there is no error, and returns an error otherwise.
+
+</returns>
+
+::::
+
+:::: right
+
+```shell
+curl --request DELETE 'https://api.vyte.in/v2/vytemes/john-doe/types/lunch-or-dinner' \
+--header 'Authorization: vkjvi2bvfo54ssbybmcts0x42z1sbzm6t0mot8trh8i03reno0'
+```
+
+> RESPONSE SAMPLE
+
+```json light-code
+{
+  "n": 1,
+  "ok": 1
+}
+```
+
```

Comment:
no the response is actually `204` deleted or `404` not found

Reply:
done a004104d92e7775d157df1b7c2050653f7e7f587